### ISSUE TITLE
Line style for line charts

### ISSUE THREAD

**Issue by EmaGht** [swimlane/ngx-charts#876]
**I'm submitting a ...**  (check one with "x")

- [ ] bug report - search github for a similar issue or PR before submitting
- [X] feature request
- [ ] support request - use StackOverflow (add the `ngx-charts` tag) or the gitter chat for support questions

**Current behavior**
Line charts (as far as i could see) has only one default appearance for every line (continuous)

**Expected behavior**
Would be nice to be able to set line appearance to something like "dashed" or similar, bonus points for setting just a portion of the data as "dashed"

**What is the motivation / use case for changing the behavior?**
Have a better visual of the resulting chart, for example to recognize real values against estimated values.
Something like this: http://www.andypope.info/ngs/ng45.gif

**Comment by nikkelj** [comment]
It should also support different marker styles to overlay on the line, similar to matlab, matplotlib, etc.  Ie, small dots, asterisks, plus, circle, box, etc etc.

**Comment by nikkelj** [comment]
It should also support a way to control the size of said markers, and the lines

**Comment by DmitryEfimenko** [comment]
is there any workaround?

**Comment by comfortme** [comment]
you can make series dashed with stroke-dasharray property with css
but in order to have a series partly dashed and partly solid, you have to split series into multiple series i guess.

**Comment by jrhite** [comment]
@comfortme thanks for the tip. i'm just looking to style a line completely as dashed, but if I have a series of 4 lines and need only one of them to be dashed which selector would I use to have stroke-dasharray applied to that specific line I want dashed?

**Comment by comfortme** [comment]
hmm :nth-child maybe?

**Comment by ljisman** [comment]
@jrhite  Any update on this?

**Comment by jrhite** [comment]
I used a combination of the two tips given to me above, `stroke-dasharray` property with `nth-child` by comfortme. It worked well.

    // here 3 is the specific line I want to be dashed
    g.line-chart > g:last-of-type > g:nth-child(3) g.line-series > path {
        stroke: #72b4ec;
        stroke-dasharray: 6 10;
    }

**Comment by lukekroon** [comment]
@jrhite where did you put that css to change the line chart ?

**Comment by chriszrc** [comment]
I used a similar css approach for getting other shapes besides circles in the bubble chart, but it still feels very hacky. Seems like a very common requirement to be able to add a css class to lines or points in a series, seems like this could really be a powerful change that could enable a lot more chart flexibility-

https://github.com/swimlane/ngx-charts/issues/998#issuecomment-503569946

**Comment by gunveer-ppp** [comment]
Any update on setting just a portion of the line data as "dashed"? or any workaround available for this task?

**Comment by JUBINBIJU** [comment]
> I used a combination of the two tips given to me above, `stroke-dasharray` property with `nth-child` by comfortme. It worked well.
> 
> ```
> // here 3 is the specific line I want to be dashed
> g.line-chart > g:last-of-type > g:nth-child(3) g.line-series > path {
>     stroke: #72b4ec;
>     stroke-dasharray: 6 10;
> }
> ```
Hai,
where did you add this code, in the global style.scss?
Cuz I have been trying to achieve this in different ways but nothing seems to work out.

**Comment by chriszrc** [comment]
@JUBINBIJU yes, in the global style.scss - "I had to put it in the global scss file" - https://github.com/swimlane/ngx-charts/issues/876#issuecomment-504464766

**Comment by JUBINBIJU** [comment]
> @JUBINBIJU yes, in the global style.scss - "I had to put it in the global scss file" - [#876 (comment)](https://github.com/swimlane/ngx-charts/issues/876#issuecomment-504464766)


@chriszrc I added it in style.scss but still the chart remains the same. What might be the possible issue?
This is the chart i made with data that makes 3 lines
        <div class="chart-container">
            <ngx-charts-line-chart
                [view]="[500, 400]"
                [results]="lineChartDataset"
                [xAxisLabel]="'x-axis'"
                [yAxisLabel]="'y-axis'"
                [legend]="true"
                [showXAxisLabel]="true"
                [showYAxisLabel]="true"
                [xAxis]="true" 
                [yAxis]="true"> 
            </ngx-charts-line-chart>
        </div>

**Comment by chriszrc** [comment]
@JUBINBIJU add a codesandbox, impossible to tell without it, you can start off something like this - https://codesandbox.io/s/angular-ngx-charts-e7hx2
